app/docs/0.11.x/getting-started/quickstart.md

```diff
@@ -20,7 +20,8 @@ supports PostgreSQL and Cassandra).
     Issue the following command to [start][CLI] Kong:
 
     ```bash
```

Comment:
The same in all the other changes -two separate command prompts instead of one with a comment.